compiler/src/dmd/attrib.h

```diff
@@ -32,7 +32,6 @@ class AttribDeclaration : public Dsymbol
     bool oneMember(Dsymbol *&ps, Identifier *ident) override;
     bool hasPointers() override final;
     bool hasStaticCtorOrDtor() override final;
-    void checkCtorConstInit() override final;
```

Comment:
You can't simply delete these methods since they seem to be used by 3rd party compilers. You will notice that at the beginning of the file there is a "namespace dmd" declaration. That is where you should move this function.

Same for the other declarations from header files.

Reply:
It is unused directly by LDC and a [cursory search](https://github.com/search?q=repo%3AD-Programming-GDC%2Fgcc%20checkCtorConstInit&type=code) reveals no uses by GDC.